gnomad_qc/analyses/compute_pext.py

```diff
@@ -224,6 +225,20 @@ def main(args):
     ht = pext_res.context_ht.ht().naive_coalesce(5000)
     tx_mt = pext_res.tx_mt.mt()
 
+    # Any tissue below 50 samples will NOT get shown on website in overall pext or as
```

Comment:
should 50 be 100 here based on what you sent?
> We observed that for low number of samples the variation in the number of expressed genes is
higher. The number of expressed genes reaches saturation for sample numbers over 100. This
suggests that for tissues with low number of available samples: kidney cortex, kidney medulla,
bladder, endocervix, ectocervix, and fallopian tube the analyses (correlation of gene expression
between tissues, number of expressed genes, number of expressed transcripts) may display high
variation.

https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-08125-9

Reply:
Yeah, it's 100 based on what I sent, but Konrad wanted to lower it to 50 to rescue Bladder in v10.